scripts/pcm-dpc/covid-19/dpc-covid19-ita_StatisticalVariable.mcf

```diff
@@ -0,0 +1,95 @@
+Node: dcid:count_MedicalConditionIncident_COVID_19_HospitalizedsWithSymptoms
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:HospitalizedsWithSymptoms
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_IntensiveCare
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:IntensiveCare
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_Hospitalized
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:Hospitalized
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_PeopleInHomeIsolation
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PeopleInHomeIsolation
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_ActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:ActiveCase
+
+Node: dcid:incrementalCount_MedicalConditionIncident_COVID_19_IncrementalActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+statType: dcs:measuredValue
+medicalStatus: dcs:IncrementalActiveCase
+
+Node: dcid:cumulativeCount_MedicalConditionIncident_COVID_19_Recovered
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:Recovered
+
+Node: dcid:cumulativeCount_MedicalConditionIncident_COVID_19_Death
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:Death
+
+Node: dcid:cumulativeCount_MedicalConditionIncident_COVID_19_PositiveCase
```

Comment:
Reuse: https://github.com/datacommonsorg/data/blob/master/scripts/covid_tracking_project/historic_state_data/COVIDTracking_States_StatisticalVariables.mcf#L9

Reply:
Is this based on number of people or number of test? I hesitate to use Medical Test for positive cases, because I think we are assuming the test is based on number of tests, while my data is based on number of people.